Add Multi preference zone to yb-admin (#12341)
yugabyte/yugabyte-db: docs/content/preview/admin/yb-admin.md

#### set_preferred_zones

> Sets the preferred availability zones (AZs) and regions. Tablet leaders are placed in alive and healthy nodes of the highest preference zones (preference 1). When no such node is available, then alive and healthy nodes from the next highest preferred zone are picked. Zones with no preference are equally eligible to host tablet leaders.

Reviewer: Good clarification. Instead of saying "highest preference", perhaps re-word as "most preferred" ?  Something like .. "are placed in alive and healthy nodes in availability zone in order of preference" ?

Reviewer 2: Tablet leaders are placed in alive and healthy nodes of AZs in order of preference. When no healthy node is available in the preferred AZ, then alive and healthy nodes from the next preferred zone are picked. Zones with no preference are equally eligible to host tablet leaders.

{{< note title="Note" >}}

* Make sure you've already run [`modify_placement_info`](#modify-placement-info) command beforehand.

> * Having all tablet leaders reside in a single region will reduce the number of network hops for the database to write transactions and thus increase performance and lowering latency.

Reviewer 2: ```suggestion
* Having all tablet leaders reside in a single region reduces the number of network hops for the database to write transactions, which increases performance and lowers latency.
```

> * Having all tablet leaders reside in a single region will reduce the number of network hops for the database to write transactions and thus increase performance and lowering latency.

Reviewer: lowers --> reduces

> * By default, the transaction tablet leaders will not respect these preferred zones and will be balanced across all nodes. In the transaction path, there is a roundtrip from the user to the transaction status tablet serving the transaction - if the leader closest to the user is used rather than forcing a roundtrip to the preferred zone, then there will be efficiency improvements.

Reviewer 2: ```suggestion
* By default, the transaction tablet leaders don't respect these preferred zones and are balanced across all nodes. Transactions include a roundtrip from the user to the transaction status tablet serving the transaction - using the leader closest to the user rather than forcing a roundtrip to the preferred zone improves performance.
```

> * Leader blacklisted nodes will not host any leaders irrespective of their preference.

Reviewer 2: ```suggestion
* Leader blacklisted nodes don't host any leaders irrespective of their preference.
```

> * When picking leaders the order of precedence is placement information, leader blacklist and finally preferred zones.

Author: Will rewrite this as:

Leader blacklist negates the preference.

> * affinitized_leaders in cluster configuration may be replaced with multi_affinitized_leaders.

Reviewer 2: what are affinitized leaders? not previously described anywhere

Author: Will switch to 
"Cluster configuration stores preferred zones in either affinitized_leaders or multi_affinitized_leaders object"

> * Tablespaces do not inherit cluster level placement information, leader preference and read replica configurations.

Reviewer 2: ```suggestion
* Tablespaces don't inherit cluster-level placement information, leader preference, or read replica configurations.
```
